packages/flame_forge2d/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # CHANGELOG
 
-## [next]
+## [Next]
  - Update mechanism by which `BodyComponent`'s are disposed to use the `onRemove` method
+ - Flip y-axis after translation of body position, so that normal flame components can be children
+ - Update to Forge2D 0.8.0
+ - Update to Flame 1.0.0-releasecandidate.13
```

Comment:
Do we have a rc13?

Reply:
This one can't be released until we have rc13 since it depends on the `postRender` method.

Reply:
Right! I wasn't thinking in the 4th dimension!

![brown](https://user-images.githubusercontent.com/835641/124763694-a375b780-df0a-11eb-8cfe-3b50ee72089a.jpeg)